model/uzivatel.php

```diff
@@ -654,6 +654,19 @@ public function jmenoNick()
         return self::jmenoNickZjisti($this->r);
     }
 
+    /**
+     * Novější formát zápisu jména a příjmení uživatele ve tvaru Jméno Příjmení (Nick).
+     * Funkce uvažuje možnou absenci nicku.
+     */
+    public function jmenoVolitelnyNick()
+    {
+        if ($this->nick()) {
+            return $this->jmeno() . ' (' . $this->r['login_uzivatele'] . ')';
```

Comment:
Kontrolujeme **nick**, abychom použili **login** ? To je dost kriptické.

Buďto bychom měli používat všude jedno, nebo druhé.

Reply:
Dobrý postřeh, určitě to jde čitelněji :+1: